### ISSUE TITLE
how to export jimple to Jar?

### ISSUE THREAD

**Issue by colordancer** [secure-software-engineering/CodeInspect#204]
It's possible to import a Jar package and decompile it into jimple to modify. 
But how to export jimple to Jar? I could not find any option with in export menus.

**Comment by MarcMil** [comment]
At the moment, as the noticed, there is no GUI option to export it yet.
However, CodeInspect automatically generates one. To find it, open the context menu on the project, click on Properties.
You can see the location of the project on disk at the "Location:" label.
Go to the specified location of the project on the disk. There you can find an Output.jar file.

Also, you should be able to see the Output.jar in the Project Explorer.

**Comment by StevenArzt** [comment]
Let's take this as a feature request. I added the respective label. It shouldn't be too hard to add an option into the export dialog that copies the output.jar file to a location specified by the user. This is much more convenient than the current solution.

**Comment by colordancer** [comment]
@MarcMil cheers. What I did then is just the way you mentioned. But AFAIK, the quality of output.jar is not that good.

**Comment by MarcMil** [comment]
@colordancer What do you mean by "the quality is not that good"? I just realized it contains some .tmstamp files; I have fixed that now. Apart from that, the zip file should contain all relevant entries from the original file.
Do you mean that the compression ratio is not as good as with the imported jar?

**Comment by colordancer** [comment]
@MarcMil Thanks. Quality means two parts. One is the junk .tmstamp files. The other is that the re-compiled Jar package seems not working as well as the original package. I'm not sure I got the output.jar at the right time, as I refreshed the project structure once I modified something. But even though, I found there were something different between output.jar and the imported jar. If it helps, I could send you the two Jar packages I tested.
